BasicGates/Workbook_BasicGates.ipynb

```diff
@@ -1017,10 +1091,667 @@
+    "\n",
+    "First of all which qubit? Looking back at Task 1.9 it seems clear that we need to use qubit[0].\n",
+    "So in what order should we apply the gates? Remember that the Pauli-Zgate flips the phase of the $|1\\rangle$ component of the superposition and leaves the $|0\\rangle$ component alone.   \n",
+    "Let's experiment with applying X to qs[0] first.  first.Looking at what could now be our half way answer of $\\frac{1}{\\sqrt{2}} \\big(|01\\rangle + |10\\rangle\\big)$ we can see that if we apply the Z gate to qs[0] it will leave the $0_{A}$ alone but flip the phase of $1_{A}$ to $-1_{A}$ so flipping the phase of the $|11\\rangle$ component of our  Bell state.  \n",
+    "Let's see this in action by running the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3fb30e25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%kata T110_BellStateChange3\n",
+    "\n",
+    "operation BellStateChange3 (qs : Qubit[]) : Unit is Adj+Ctl {\n",
+    "    Z(qs[0]);\n",
+    "    X(qs[0]);\n",
+    "   \n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "330538d3",
+   "metadata": {},
+   "source": [
+    "Let's also see the effect of using these gates in the reverse order, i.e. Z then X. Run the cell below and look at the actual state."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3cd50144",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%kata T110_BellStateChange3\n",
+    "\n",
+    "operation BellStateChange3 (qs : Qubit[]) : Unit is Adj+Ctl {\n",
+    "    Z(qs[0]);\n",
+    "    X(qs[0]);\n",
+    "   \n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b345ede4",
+   "metadata": {},
+   "source": [
+    "What we have there is $\\frac{1}{\\sqrt{2}} \\big(-|01\\rangle + |10\\rangle\\big)$. This can be written as  $-\\frac{1}{\\sqrt{2}} \\big(|01\\rangle - |10\\rangle\\big)$. Thus we can see that using the gates in the reverse order compared to the correct solution has introduced a global phase of  -1."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b2ca88f1",
+   "metadata": {},
+   "source": [
+    "As a final comment, let's notice that the operation is marked as **Adj** (short for adjoint. This means that it must be reversible. Let's do an experiment to test this with our correct answer. We will apply  X followed by  Z and then reverse this and apply Z followed by X in one sequence . This sequence  should take us back to the initial state. Run the next cell and just look at the actual state produced and not the final judgement."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24ef1b32",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%kata T110_BellStateChange3\n",
```

Comment:
I think the CI build is not happy about this code cell: for the workbooks, it checks that the code provided in the cell passes the test (as opposed to the main kata notebooks, in which the CI build replaces %kata magic with %check_kata magic which, in turn, checks that the reference solution to this task is correct).
To achieve the same pedagogical effect without upsetting the CI build, you'll need to say something like "try to add lines `Z(qs[0]); X(qs[0]);`" to the previous cell and see how that changes the input state